os/drivers/pm/pm.c

```diff
@@ -129,6 +129,11 @@ static int pm_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 			}
 		}
 		break;
+#ifdef CONFIG_PM_METRICS
+	case PMIOC_METRICS:
+		ret = pm_metrics((int)arg);
+		break;
+#endif
```

Comment:
Please find the API name of pm_driver_register(), the name above the description block is wrong, please correct it

Reply:
I think, this agenda, in the https://github.com/Samsung/TizenRT/pull/6257 is better
Could you leave the comment about this  to https://github.com/Samsung/TizenRT/pull/6257 also?

Reply:
@ewoodev @edwakuwaku I have resolved it in #6257